src/api/DesktopAgent.ts

```diff
@@ -165,4 +171,11 @@ export interface DesktopAgent {
    * Returns `null` if the app is not joined to a channel.
    */
   getCurrentChannel(): Promise<Channel>;
+
+  /**
+   * Returns an Array of `Channel` objects for the current channel membership.
+   *
+   * Returns `null` if the app is not joined to any channels.
```

Comment:
As per @mattjamieson's comment, let's change this to an empty array.

Reply:
I changed the output of getCurrentChannels (plural) to an empty array. Leaving getCurrentChannel as it is